### PR TITLE
feat(entities-keys): migrate `entities-keys` from `shared-ui-components`

### PR DESCRIPTION
**Pull request by nekolab**
# Summary

## PR Checklist

* [ ] **Naming & Structure:** the files and package structure use the conventions outlined in the [Creating a Package docs](https://github.com/Kong/public-ui-components/blob/main/docs/creating-a-package.md).
* [ ] **Tests pass:** check the output of all package unit and/or component tests.
  * [ ] If this PR is the result of a bug, test coverage was added accordingly.
* [ ] **Functional:** all changes do not break existing APIs, but if so, a BREAKING CHANGE commit is in place to bump the major version.
* [ ] **Conventional Commits** all commits follow the conventional commit standards [outlined in the main README](https://github.com/Kong/public-ui-components#committing-changes).
* [ ] **Docs:** includes a technically accurate README, and the docs have been updated accordingly based on the changes in this PR.

### CODE REVIEW

File: packages/entities/entities-keys/docs/key-config-card.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+A `@navigation-click` event is emitted when a user clicks the `key set name` button and the event payload is the entity type (`key-sets`) and the key set id.
+
+### Usage example
+
+Please refer to the [sandbox](../sandbox/pages/KeyConfigCardPage.vue). The page is accessible by clicking on the row or `View details` button of an existing Key.
+
+## TypeScript interfaces
+
+TypeScript interfaces [are available here](https://github.com/Kong/shared-ui-components/blob/main/packages/entities/entities-keys/src/types/key-config-card.ts) and can be directly imported into your host application. The following type interfaces are available for import:
```
Comment (Leopoldthecoder) [inline packages/entities/entities-keys/docs/key-config-card.md:108]:
```suggestion
TypeScript interfaces [are available here](https://github.com/Kong/public-ui-components/blob/main/packages/entities/entities-keys/src/types/key-config-card.ts) and can be directly imported into your host application. The following type interfaces are available for import:
```

Could do another round of replacing `shared-ui-components` with `public-ui-components`? I see there're more instances of `shared-ui-components` in other `.md` files besides this one.

**Review by Leopoldthecoder — APPROVED** [review]
LGTM

**Comment by Leopoldthecoder** [comment]
Verified [here](https://github.com/Kong/kong-admin/pull/2841/checks).

**Comment by nekolab** [comment]
Verified [here](https://github.com/Kong/kong-admin/actions/runs/5816606985)~

**Pull request merged**